happi/cli.py

```diff
@@ -237,7 +237,7 @@ def search_parser(
               'Provide the name of the item to clone.')
 @click.pass_context
 def add(ctx, clone: str):
-    """Add new entries interactively."""
+    """Add new entries or copy existing entries."""
```

Comment:
Is it worth having an additional top-level alias `copy` (= `add --clone`) to avoid confusion?

Reply:
I was looking into a way to make click redirect a `happi copy` command to the `happi add --clone` help message, but got side tracked.  I like the idea

Reply:
added `happi copy` in [660ed1c](https://github.com/pcdshub/happi/pull/294/commits/660ed1cc4e85227617b635628a9a0edc061d66dd)